wikimedia/wikipedia-ios
# Replace SF Symbol checkmark with PDF asset

**Phabricator:** https://phabricator.wikimedia.org/T288658 (in part)

### Notes
I missed the selection checkmark when requesting PDF assets for the Notifications Center. I believe this concludes the feedback we received on `T288658`.

### Test Steps
1. Update a cell's display state: `let displayState = NotificationsCenterCellDisplayState.editSelectedUnread`
2. Confirm the checkmark renders with expected metrics in the Notifications Center
